setup.py

```diff
@@ -40,7 +40,7 @@
     install_requires=[
         "flake8 >= 3.2.1",
         "pyflakes >= 2.1.1",
-        'ast-decompiler >= 0.4.0, <0.5.0; python_version < "3.9"',
+        'ast-decompiler >= 0.6.0, <0.7.0; python_version < "3.9"',
```

Comment:
Should we just make this `<1.0`? I doubt I'll make changes that break flake8-pyi